include/multiphase/private/fd_operators_inc.h

```diff
@@ -509,6 +509,58 @@ applyCoincompressibility(SAMRAI::tbox::Pointer<SAMRAI::hier::Patch<NDIM>> patch,
         (*A_data)(idx) = D * (div_un_thn + div_us_ths);
     }
 }
+
+inline void
+preconditonerBlockGTGOperatorConstantCoefficient(SAMRAI::tbox::Pointer<SAMRAI::hier::Patch<NDIM>> patch,
```

Comment:
This function can be ignored because we are using PoissonSolverManager to set things up.